app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime

```

Comment:
Add a blank line between the different import groups
https://peps.python.org/pep-0008/#blank-lines